### PR TITLE
Revert "No more message spam when you witness someone being attacked or shot"

### PR DESCRIPTION
**Pull request by ghost**
Reverts tgstation/tgstation#20962

Reasoning-Feedback has been overwhelmingly negative and it can be hard to tell who's doing what leading to giant fuckpits of rage, or even being able to tell if your own attacks are hitting something such as a spiderling since no message pops up.

While the actual negative downsides (X IS HIT BY THE LASER x10) still shows up, meaning it's entirely negative with no positives.

babby's first revert please do not hurt me

### CODE REVIEW

File: code/_onclick/item_attack.dm [hunk 1]
```diff
@@ -56,8 +57,7 @@
 
 /obj/attacked_by(obj/item/I, mob/living/user)
 	if(I.force)
-		visible_message("<span class='danger'>[user] has hit [src] with [I]!</span>", null, null, 2, user)
-		//only witnesses close by and the victim see a hit message.
+		user.visible_message("<span class='danger'>[user] has hit [src] with [I]!</span>", "<span class='danger'>You hit [src] with [I]!</span>")
```
Comment (ChangelingRain) [inline code/_onclick/item_attack.dm]: All you needed to do was readd the 'hit with x' as the second argument, honestly.

**Comment by ChangelingRain** [comment]
This is very, very clumsy and reverts some good/useful code just to readd attack messages, when it would be extremely simple to just readd them manually without reverting the code.

**Comment by AdamElTablawy** [comment]
yes, yes it is

however i did not know that at the time, i just pressed the 'revert' button

i suppose I can just do what you suggested but again i'm no coder so it's liable to just be a horrific mess

**Comment by Cheridan** [comment]
no butthurt reverts

**Comment by AdamElTablawy** [comment]
oi can i have an actual reason beyond 'butthurt reverts' seeing as a majority of people don't like it

File: code/_onclick/item_attack.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/_onclick/item_attack.dm]: actually slightly more complex but hey whatever i'll fix this!

**Comment by Cheridan** [comment]
no

**Comment by AdamElTablawy** [comment]
why

**Comment by Crushtoe** [comment]
Because fuck you this is devbus territory.

**Pull request closed without merging**